tests/test_utils.py

```diff
@@ -303,3 +306,437 @@ def check_multistream_classic(fnames):
                     actual, expected, decimal=4,
                     err_msg='Variable=%s, freq=%s, how=%s: '
                             'failed comparison' % (key, freq, how))
+
+def tsplit(string, delimiters):
+    '''Behaves like str.split but supports multiple delimiters.
+
+    '''
+
+    delimiters = tuple(delimiters)
+    stack = [string,]
+
+    for delimiter in delimiters:
+        for i, substring in enumerate(stack):
+            substack = substring.split(delimiter)
+            stack.pop(i)
+            for j, _substring in enumerate(substack):
+                stack.insert(i+j, _substring)
+
+    return stack
+
+def plot_science_tests(driver, testname, science_test_data_dir, result_dir, plot_dir, vic_42_dir, vic_50_dir,
+                        obs_dir, plots_to_make):
+
+    ''' makes science test figures
+
+    Parameters
+    ----------
+    driver: <str>
+        Name of Driver
+    testname: <str>
+        Name of test
+    science_test_data_dir: <str>
+        Science test data directory
+    result_dir: <str>
+        Result directory
+    plot_dir: <str>
+        Directory for output plots
+    vic_42_dir: <str>
+        Directory for VIC 4.2 archive
+    vic_50_dir <str>
+        Directory for VIC 5.0 archive
+    obs_dir <str>
+        Directory for observations archive
+    plots_to_make <Dict>
+        Keys that indicate which plots should be made
+
+    Returns
+    ----------
+    '''
+    if testname == "science_test_snotel":
```

Comment:
I think it would be better if we used `test_type` instead of `testname`.  I could image a case where we want to have multiple snotel science tests using different parameters that both need `plot_snotel_comparison()`.  This would require a minor addition to the science test configuration file.